CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # webkms-client ChangeLog
 
+## 4.0.0 -
+
+### Changed
```

Comment:
Also, dropping support for node versions should be listed here.  Since node 10 is EOL in april this year, in other places we're going ahead and dropping support for node 10 as well.

https://nodejs.org/en/about/releases/

Reply:
Sure, will remove.

Reply:
Added